pkg/ui/workspaces/cluster-ui/src/statementsDiagnostics/activateStatementDiagnosticsModal.tsx

```diff
@@ -293,8 +293,10 @@ export const ActivateStatementDiagnosticsModal = React.forwardRef<
               disabled={!expires}
               value={expiresAfter}
               onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
-                if (parseInt(e.target.value) > 0) {
+                if (!isNaN(parseInt(e.target.value))) {
```

Comment:
It seems like this change will allow for negative numbers. Is this something we want to support?